### PR TITLE
Do not forget message to ack if we send out "just" a Standalone ack

### PR DESCRIPTION
**Pull request by Apollon77**
When we send out a Standalone Ack because of a Duplicate (or close) we should not "forget" the message which is still in transit and waits to be acked officially, so not clear it.

### CODE REVIEW

**Comment by Apollon77** [comment]
Ok, was produce weird side effects ,... need to dig deeper

**Comment by Apollon77** [comment]
I open again ... the issues came because Alexa did strange things and tried to commission the device from two echo devices in parallel but then stopped responding on one of em, so in fact errors I saw in log were correct. i will test further but it still seems to be right

File: src/matter/common/MessageExchange.ts [hunk 1]
```diff
@@ -99,7 +99,11 @@ export class MessageExchange<ContextT> {
         if (messageId === this.receivedMessageToAck?.packetHeader.messageId) {
             // Received a message retransmission but the reply is not ready yet, ignoring
             if (requiresAck) {
-                await this.send(MessageType.StandaloneAck, new ByteArray(0));
+                try {
+                    await this.send(MessageType.StandaloneAck, new ByteArray(0));
+                } catch (error) {
```
Comment (mfucci) [inline src/matter/common/MessageExchange.ts:104]: Can you remove this catch here? Should be catch at top level

**Review by mfucci — APPROVED** [review]

File: src/matter/common/MessageExchange.ts (hunk 1 above)
Comment (Apollon77) [reply src/matter/common/MessageExchange.ts:104]: done

Comment (Apollon77) [reply src/matter/common/MessageExchange.ts:104]: (and checked, yes should be catched elsewhere)

**Pull request merged**